### ISSUE TITLE
FutureWarning: iteritems is deprecated

### ISSUE THREAD

**Issue by stefanbischof** [cadmiumkitty/rdfpandas#16]
When using rdfpandas the following warning is shown:

```
.../site-packages/rdfpandas/graph.py:41: FutureWarning: iteritems is deprecated and will be removed in a future version. Use .items instead.
  for (column, value) in series.iteritems():
```

I guess this originates from the current version of pandas. I get a similar error then for openpyxl:

```
.../site-packages/openpyxl/worksheet/_reader.py:312: UserWarning: Data Validation extension is not supported and will be removed
  warn(msg)
```

**Comment by cadmiumkitty** [comment]
Hi Stefan, thanks for reporting this and sorry it took forever to come back. I'll try and tackle it at the same time as addressing the CURIE issue https://github.com/cadmiumkitty/rdfpandas/issues/15

**Comment by cadmiumkitty** [comment]
Fixed in release 1.1.6.
